infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -193,8 +193,8 @@ public SequencerServer(ServerContext serverContext) {
     * @param conflictParam The conflict parameter.
     * @return A conflict hash code.
     */
-    public String getConflictHashCode(UUID streamId, int conflictParam) {
-        return streamId.toString() + Integer.toString(conflictParam);
+    public String getConflictHashCode(UUID streamId, byte[] conflictParam) {
+        return streamId.toString() + Utils.bytesToHex(conflictParam);
```

Comment:
Yeah no. Using a byte[] as a key would not by trivial. You're free to attempt to implement it, but I warn you, it'll involve generating a bytebuffer to create the key, then putting the byte[] into a custom wrapper class because caffeine doesn't support custom comparators.